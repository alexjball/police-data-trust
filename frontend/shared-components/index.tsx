
import ResponseTextArea from './response-textarea/response-textarea'
import EnrollmentCTA from './enrollment-cta/enrollment-cta'
import EnrollmentHeader from './enrollment-header/enrollment-header'
import EnrollmentInput from './enrollment-input/enrollment-input'
import FormLevelError from './error-form-level/error-form-level'
import InfoTooltip from './info-tooltip/info-tooltip'
import Logo from './logo/logo'
import USAStateInput from './state-input/state-input'

export {
  ResponseTextArea,
  EnrollmentCTA,
  EnrollmentHeader,
  EnrollmentInput,
  FormLevelError,
  InfoTooltip,
  Logo,
  USAStateInput
}