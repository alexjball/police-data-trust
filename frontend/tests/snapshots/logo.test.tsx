import * as React from "react"
import { Logo } from "../../shared-components"
import renderer from "react-test-renderer"

it("renders Logo correctly", () => {
  const tree = renderer.create(<Logo />).toJSON()
  expect(tree).toMatchSnapshot()
})
