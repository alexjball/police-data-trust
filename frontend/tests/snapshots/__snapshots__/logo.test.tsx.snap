// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo correctly 1`] = `
<img
  alt="A blue circular logo design reads 'National Police Data Coalition'"
  className="logo medium"
  src="/NPDCLogo.svg"
/>
`;
